cost/google/idle_compute_instances/google_idle_compute_instances.pt

```diff
@@ -262,10 +289,10 @@ policy "pol_utilization" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing Google instance StackDriver Utilization data"
     detail_template <<-EOS
 # Google Instance StackDriver Utilization
-| Region | instanceID | Hostname | CPU Maximum % | CPU Minimum % | CPU Average % | Memory Minimum % | Memory Maximum % | Memory Average % | Labels |
-| ------ | ---------- | -------- | ------------- | ------------- | ------------- | ---------------- | ---------------- | ---------------- | ------ |
+| Project ID | Region | instanceID | Hostname | CPU Maximum % | CPU Minimum % | CPU Average % | Memory Minimum % | Memory Maximum % | Memory Average % | Labels |
```

Comment:
please sort by projectid and region

Reply:
I have addressed this comment please review the changes.